tests/test_distance.py

```diff
@@ -37,6 +37,11 @@ def facilities_data():
     )
     return fac_gdf
 
+@pytest.fixture
+def road_network():
+    # I need the road network and cannot just hand-make that data.
+    return pickle.load(open("tests/test_assets/road_network_TLS.pkl", "rb"))
```

Comment:
This is obviously not ideal as we would need a test_assets folder with the pkl file (I did not push it to git yet because it is quite large ~15Mb). But, the method needs the road network and I don't know how to create that by hand. The one linked here is created from the get_road_network method and then saved as a pickle file.

Reply:
Is there no way to create a very simple road network? E.g. using an LLM do do the grunt work for you? 😅

Reply:
We discussed it :)

Reply:
I will also need a fake road network for another unit test, btw. So either we can work together or I'll reuse whatever you come up with :)